src/org/usfirst/frc4904/sovereignty/FusibleNavX.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.standard.custom.sensors.NavX;
+import edu.wpi.first.wpilibj.SerialPort.Port;
+
+public class FusibleNavX extends NavX implements Fusible<Double> {
```

Comment:
I think this is fine for now, since the NavX is a large conglomeration of sensors and any one of them could be returned in `getValue()`. In mid-season we will generalize this.